### PR TITLE
End the new wizard meta

### LINKED ISSUE #14982: Suicide + mindswap
Its a thing now I guess. A player was able to suicide then mindswap in quick succession leaving the target dead

### PR DESCRIPTION
**Pull request by neersighted**
Prevent mindswap when suiciding

Fixes #14982

### CODE REVIEW

**Comment by Incoming5643** [comment]
I don't really understand how this was happening, suicde is supposed to be instant now. A one life tick trick?

**Comment by Thunder12345** [comment]
There's a delay before killing yourself with gun suicides

**Comment by Fox-McCloud** [comment]
:+1: 

This is a jackass move, anyway.

File: code/datums/spells/mind_transfer.dm [hunk 1]
```diff
@@ -43,6 +43,10 @@ Also, you never added distance checking after target is selected. I've went ahea
 		user << "<span class='warning'>They appear to be catatonic! Not even magic can affect their vacant mind.</span>"
 		return
 
+	if(user.suiciding)
+		user << "<span class='warning'>You're killing yourself! You can't concentrate enough to do this!</span>"
```
Comment (neersighted) [inline code/datums/spells/mind_transfer.dm]: Should really be "STOP TRYING TO EXPLOIT THE GAME YOU SHITLORD!"
And then `gib()`.

Comment (Fox-McCloud) [inline code/datums/spells/mind_transfer.dm]: "the wizard federation is most displeased with your dishonorable acts and has formally decided to smite you"

dust()

**Pull request merged**